scripts/paper/phase44b_public_projection_logup_probe.py

```
@@ -0,0 +1,1140 @@
+    for field in (
+        "kv_history_chunk_size",
+        "kv_history_sealed_chunks",
+        "kv_history_open_chunk_pairs",
+        "kv_history_frontier_pairs",
+        "lookup_transcript_entries",
+        "lookup_frontier_entries",
+    ):
+        usize_base(f"{label}.{field}", state[field])
+
+
+def verify_demo_trace(trace: dict[str, Any]) -> None:
```

Ruff (0.15.10)

[warning] Too many branches (22 > 12)

(PLR0912)